src/features/Chat/index.tsx

```diff
@@ -14,16 +14,19 @@ import WelcomeWindow from './components/WelcomeWindow';
 
 const Chat = () => {
   const { isTablet } = useGetLayoutMode();
-  const chat = useAppSelector(selectActiveChat);
+  const isActiveChat: boolean = useAppSelector(selectIsActiveChat);
+  const anyChats: boolean = useAppSelector(selectAnyChats);
```

Comment:
This is boolean, I would rename the selector to `selectIsChatsExisting` and this variable to `isChatsExisting` > also remove the explicit type annotation

Reply:
I renamed them to `selectChatsExist` and `selectActiveChatExists`. What do you think?

Reply:
Yeah, I think it is good!